src/contracts/AccessControlLP.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {MultiLP} from "./MultiLP.sol";
+
+abstract contract AccessControlLP is MultiLP {
```

Comment:
We'll also need a boolean to toggle the liquidityProviderCaps checks on and off. No need to pack or do anything fancy.